### ISSUE TITLE
Should not throw error on unknown attribute

### ISSUE THREAD

**Issue by thijskh** [OpenConext/OpenConext-profile#55]
When a user has an attribute that is unknown to profile (or: to stepup-saml-bundle), it throws a 500 error into the user's face.

```
Aug 22 19:15:09 papp3 openconext-profile[25780]: request.CRITICAL: Uncaught PHP Exception Surfnet\SamlBundle\Exception\UnknownUrnException: "Urn "urn:schac:attribute-def:schacPersonalUniqueCode" has not been defined in the AttributeDictionary" at /opt/openconext/OpenConext-profile-0.3.0/vendor/surfnet/stepup-saml-bundle/src/SAML2/Attribute/AttributeDictionary.php line 130 {"exception":"[object] (Surfnet\\SamlBundle\\Exception\\UnknownUrnException(code: 0): Urn \"urn:schac:attribute-def:schacPersonalUniqueCode\" has not been defined in the AttributeDictionary at /opt/openconext/OpenConext-profile-0.3.0/vendor/surfnet/stepup-saml-bundle/src/SAML2/Attribute/AttributeDictionary.php:130)"} []
```

Just adding this specific attribute to the AttributeDictionary is not sufficient. Since we pass all attributes we get from the IdP on to profile, there can be any attribute that happens to be unknown to us in the set and it's not friendly to error on that. Instead, the attribute should probably just be presented in the list, but instead of with a "nice" name, with its actual name.

**Comment by arothuis** [comment]
A solution in the Stepup-saml-bundle will offer Profile a way to allow fallbacks in the created AttributeSet. 

An attribute consists of a definition (a name, urn:mace and urn:oid) and a value. How should we deal with fallback attributes? We could create a definition during runtime for the given attribute: setting the same name, urn:mace and urn:oid as the received attribute. This could possibly result in showing an attribute as an urn:mace and an urn:oid if it's present in the Assertion as both an urn:mace as an urn:oid, while other (known) attributes are deduplicated using the dictionary.

As we barely have information about the given attribute, it would be difficult to match its `urn:mace` with an `urn:oid`. Therefore, it would make sense to just show them both and pass on deduplicating them. However, this would not be consistent with the other attributes that are deduplicated.

@thijskh, what do you think?

**Comment by thijskh** [comment]
Indeed, if an attribute is unknown we cannot do much optimization. So it's fine to show both `urn:mace` and `urn:oid` in case of an unknown attribute.

**Comment by arothuis** [comment]
Implemented in #51 and #68
